README.md

```diff
@@ -141,7 +147,7 @@ Patches are encouraged, and may be submitted by forking this project and submitt
 
 ## License
 
-Copyright 2014 Google, Inc.
+Copyright 2015 Google, Inc.
```

Comment:
We should confirm with OSPO, but I've heard that we might actually be able to remove the year from copyright statements, which helps avoid doing an annual copyright PR.